ext/stringio/stringio.c

```diff
@@ -392,11 +401,24 @@ strio_finalize(VALUE self)
 }
 
 /*
- * call-seq: StringIO.open(string=""[, mode]) {|strio| ...}
+ * call-seq:
+ *   StringIO.open(string = '', mode = 'r+') {|strio| ... }
+ *
+ * Creates a new \StringIO instance formed from +string+ and +mode+;
```

Comment:
`mode` is defaulted to `r` if `string` is frozen.
Is this worth to describe?

Reply:
Added to ::new and ::open..